### PR TITLE
Adds a qa env check

### PR DESCRIPTION
**Pull request by ricardo-quinones**
### Motivation

For apps in kubernetes, there is a `qa` environment.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -25,6 +25,12 @@ export const ci = 'testCI';
 export const development = 'development';
 
 /**
+ * Qa NODE_ENV
```
Comment (nason) [inline src/constants.js]: `APP_ENV` or `NODE_ENV`? There are cascading consequences to *not* having NODE_ENV=production. 

The docs all have this as APP_ENV=qa, which seems right to me

File: src/utils.js [hunk 2]
```diff
@@ -69,6 +70,14 @@ export const isCI = (config = {}) => getEnv(config) === ci;
 export const isTest = (config = {}) => getEnv(config) === test || isCI(config);
 
 /**
+ * Is the application a qa deployment?
+ * @param  {Object} config
+ * @param  {String} config.APP_ENV
+ * @return {Boolean}
+ */
+export const isQa = (config = {}) => getStage(config) === qa;
```
Comment (nason) [inline src/utils.js]: This is a complete nit but since QA is an acronym, can we use `isQA`, `QA`, etc? See `isCI`

File: package.json [hunk 3]
```diff
@@ -79,6 +79,7 @@
     "karma-sauce-launcher": "1.1.0",
     "karma-sourcemap-loader": "0.3.7",
     "karma-webpack": "1.8.0",
+    "lodash": "4.x",
```
Comment (nason) [inline package.json]: Is this used anywhere?

File: package.json [hunk 4]
```diff
@@ -91,6 +92,7 @@
     "semver": "5.3.0",
     "sinon": "1.17.7",
     "sinon-chai": "2.8.0",
-    "webpack": "1.14.0"
+    "webpack": "1.14.0",
+    "yarn-run-all": "3.1.1"
```
Comment (nason) [inline package.json:95]: Is this used anywhere either?

Comment (ricardo-quinones) [reply package.json:95]: [Here](https://github.com/wework/env-universal/pull/78/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R28).

File: src/constants.js (hunk 1 above)
Comment (ricardo-quinones) [reply src/constants.js]: I just copy pasted from the wrong place. I'll change.

File: src/utils.js (hunk 2 above)
Comment (ricardo-quinones) [reply src/utils.js]: Sure.

File: package.json (hunk 3 above)
Comment (ricardo-quinones) [reply package.json]: For whatever reason when `yarn install`ing all the dependencies, I was getting a v3 version of lodash. Couldn't reproduce again, so I removed. `¯\_(ツ)_/¯`

**Review by nason — APPROVED** [review]

**Pull request merged**